<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWidget</class>
 <widget class="QWidget" name="StatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ms/f:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_milliseconds_per_frame">
     <property name="toolTip">
      <string>Milliseconds per Frame</string>
     </property>
     <property name="text">
      <string>t</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Samples per Pixel</string>
     </property>
     <property name="text">
      <string>s/p:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_samples_per_pixel">
     <property name="toolTip">
      <string>Samples per Pixel</string>
     </property>
     <property name="text">
      <string>t</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Rays per Second</string>
     </property>
     <property name="text">
      <string>r/s:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_rays_per_second">
     <property name="toolTip">
      <string>Rays per Second</string>
     </property>
     <property name="text">
      <string>t</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>rays:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_ray_count">
     <property name="text">
      <string>t</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>pass:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_pass_count">
     <property name="text">
      <string>t</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>304</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
